src/js/components/Ballot/MeasureItemCompressed.jsx

```diff
@@ -325,7 +328,7 @@ export default class MeasureItemCompressed extends Component {
           {/* Positions in Your Network and Possible Voter Guides to Follow */}
           <div className="u-flex u-flex-auto u-flex-row u-justify-between u-items-center u-min-50">
             {/* <ItemSupportOpposeRaccoon ballotItemWeVoteId={measureWeVoteId}
-                                    ballot_item_display_name={ballot_item_display_name}
+                                    ballotItemDisplayName={ballotItemDisplayName}
```

Comment:
I would roll this change back until we are ready to update ItemSupportOpposeRaccoon, since the prop in that file is still `ballot_item_display_name`.